.coderabbit.yml

```
@@ -0,0 +1,43 @@
+  - markdown
+
+tools:
+  java:
+    enabled: true
+    include_patterns:
+      - "src/**/*.java"
+    review_depth: thorough
+    security_focus: high
+    performance_analysis: true
+    best_practices: true
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)